ocd/src/collection/mod.rs

```diff
@@ -0,0 +1,15 @@
+pub mod cores;
+pub mod targets;
+
+use crate::target::Target;
+
+pub fn get_target(name: impl Into<String>) -> Option<Target> {
```

Comment:
`name` should be `impl AsRef<str>`, since the function doesn't need ownership

Reply:
Ohhh finally I know how you do it properly! I never managed that :))